channeldb/graph.go

```diff
@@ -1696,26 +1795,55 @@ func delChannelByEdge(edges, edgeIndex, chanIndex, zombieIndex,
 // determined by the lexicographical ordering of the identity public keys of
 // the nodes on either side of the channel.
 func (c *ChannelGraph) UpdateEdgePolicy(edge *ChannelEdgePolicy) error {
-	return c.db.Batch(func(tx *bbolt.Tx) error {
-		return updateEdgePolicy(tx, edge)
+	c.cacheMu.Lock()
```

Comment:
Also need to refresh cache after calls to `MarkEdgeLive`? Any other methods that can alter the any of the cached information?

Reply:
good cache! i did a scan of all the methods when first implementing, wouldn't hurt to do another to be safe

Reply:
`MarkEdgeZombie` also needed to be modified, fixed